### DISCUSSION TITLE
네이버페이 추가/수량 클레임 변경 대한 문의입니다.

### DISCUSSION THREAD

**Discussion by limchangtaek** [commerce-api-naver/commerce-solutions#86] [묻고 답하기]
안녕하세요. 굿스플로입니다.

네이버페이 추가/수량 클레임 변경 건이 언제 적용되는지 문의 드립니다.

감사합니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

> [!IMPORTANT]
> 게시해주신 내용은 커머스솔루션 개발사 연동 관련 문의로 판단되어 관리자가 게시 공간을 이동하였습니다.
> - 최초 게시된 공간: 커머스API 내 스토어 애플리케이션 기술지원 공간 (`/commerce-api`)
> - 이동된 공간: 커머스솔루션 개발사 기술지원 공간 (`/commerce-solutions`)

공지사항에서의 언급 내용과 같이...
- 네이버페이 추가 클레임은 예정 기간대로 적용된 상태입니다.
- 네이버페이 수량 클레임은 네이버쇼핑 일부 버티컬 제휴사 주문건에만 적용된 상태로 2025년 중 일반 스마트스토어 판매자 대상으로 전체 확대할 계획을 갖고 있습니다.
    - https://github.com/commerce-api-naver/commerce-solutions/discussions/5
    - https://notice.naver.com/notices/cac/16069

추가되는 사항은 스마트스토어센터, 커머스API센터 공지사항으로 판매자 분들께 알려드릴 예정으로 공지사항을 주기적으로 확인해주시기 바랍니다.
더불어, 수량 클레임의 도입 이전이라도 굿스플로에서 수량 클레임 시스템 대응이 완료되었다면 아래 내용을 참고하시어 대응 가능 정보를 연동하시는 것을 권해드립니다.
- https://github.com/commerce-api-naver/commerce-solutions/discussions/59
